arch/x86_64/src/intel64/intel64_smpcall.c

```diff
@@ -60,13 +60,11 @@
 
 int x86_64_smp_call_handler(int irq, void *c, void *arg)
 {
-  struct tcb_s *tcb;
   int cpu = this_cpu();
 
-  tcb = current_task(cpu);
+  current_task(cpu);
```

Comment:
why add this

Reply:
The CI reported an error; we removed the unused code.